server/tls/tlscertificate_retriever.go

```diff
@@ -0,0 +1,20 @@
+package tls
```

Comment:
Is this windows specific file?

Reply:
No, this is not windows specific, it is the interface that will be used by both Linux and Windows using the build tags.